index.js

```diff
@@ -1,3 +1,3 @@
 module.exports = function () {
-  return false // we've been rescued!
+  return require('false') // we've been rescued!
```

Comment:
The problem is that the module `false` exports a **function** instead of a **value**, thus the requirement to call the function twice. Something like this should work:

```javascript
var untrue = require('false')

module.exports = function () {
  return untrue() // we've been rescued!
}